src/marqo/s2_inference/model_registry.py

```diff
@@ -817,6 +953,14 @@ def _get_hf_properties() -> Dict:
                  "poolingMethod": "mean",
                  "text_query_prefix": "Represent this sentence for searching relevant passages: ",
                  "notes": ""},
+            "BAAI/bge-base-en-v1.5":
+                {"name": 'BAAI/bge-base-en-v1.5',
+                 "dimensions": 768,
+                 "tokens": 512,
+                 "type": "sbert",
```

Comment:
They type should be `"hf"`

Reply:
type was sbert. Moved them under sbert method